### PR TITLE
Physical lifestyle traits

### PR DESCRIPTION
**Pull request by MemeWizard**
### Changelog:
- Added traits (5 levels each - Faintly/Slightly/Moderately/Outstandingly/Immensely) for every physical lifestyle tree.

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

## How to test:
- You can find all traits in the in-game encyclopedia and/or by spending perks in corresponding lifestyle tree.
- Check if new tooltip in physical lifestyle about traits (near the lifestyle exp progressbar) works correctly.
- Check if traits and their acquisition works correctly.

### CODE REVIEW

**Review by robmart — CHANGES_REQUESTED** [review]
Everything looks good except memory usage, which is out of control. Goes from 5 to 6.3 gigs used right when you load into the map.

image

**Comment by MemeWizard** [comment]
That's a good notice, but I'm afraid I can't reproduce it

image

**Review by robmart — APPROVED** [review]
Memory issue seems to be caused by the Fullscreen Barbershop mod, probably a weird interaction with the mod somewhere. Either way it's not the fault of this branch.

image

File: common/scripted_effects/wc_lifestyle_traits_effects.txt [hunk 1]
```diff
@@ -1,4 +1,67 @@
 ﻿wc_remove_all_magic_lifestyle_traits_effect = {
+	if = { # Endurance
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/scripted_effects/wc_lifestyle_traits_effects.txt:2]: Yep, it's intended
It's needed for dead character history and fixing https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/731#discussion_r734100152

File: common/scripted_effects/wc_lifestyle_traits_effects.txt [hunk 2]
```diff
@@ -1,4 +1,67 @@
 ﻿wc_remove_all_magic_lifestyle_traits_effect = {
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/scripted_effects/wc_lifestyle_traits_effects.txt]: Yeah, that's true
I renamed it

File: common/customizable_localization/wc_lifestyles_custom_loc.txt [hunk 3]
```diff
@@ -387,3 +387,106 @@ GetWCMagicalLifestyleElementalEarthMagicNextTraitForPerks = {
 		localization_key = "wc_elemental_earth_magic_next_trait_level_is_1"
```
Comment (zumbak04) [inline common/customizable_localization/wc_lifestyles_custom_loc.txt:387]: This branch has a lot of errors.
A few of them:
```
Line 2149: Unknown trigger type: wc_can_start_new_lifestyle_tree_duty_trigger, near line: 10
	Line 2150: Unknown trigger type: wc_can_start_new_lifestyle_tree_skulduggery_trigger, near line: 11
	Line 2151: Unknown trigger type: wc_can_start_new_lifestyle_tree_temptation_trigger, near line: 12
	Line 2152: Unknown trigger type: wc_can_start_new_lifestyle_tree_intimidation_trigger, near line: 13
	Line 2153: Unknown trigger type: wc_can_start_new_lifestyle_tree_medicine_trigger, near line: 14
	Line 2154: Unknown trigger type: wc_can_start_new_lifestyle_tree_scholarship_trigger, near line: 15
	Line 2155: Unknown trigger type: wc_can_start_new_lifestyle_tree_theology_trigger, near line: 16
	Line 2156: " in file: " file: common/lifestyle_perks/wc_shadow_magic_perks.txt line: 457; can_start_new_lifestyle_tree_trigger" near line: 42
```

This branch is to be merged to `magic_trees` and `magic_trees` is to be merged to `master`. Right now `magic_trees` has 18k changed lines. I'm pretty sure I won't check due to its size at this point. How did these errors get there? As the base and core branch of this entire feature, it must be free of such errors. Fixing them must be your top priority before requesting my reviews on child branches.

Comment (MemeWizard) [reply common/customizable_localization/wc_lifestyles_custom_loc.txt:387]: I already was answering on this question multiple times

Comment (MemeWizard) [reply common/customizable_localization/wc_lifestyles_custom_loc.txt:387]: Was fixed in #757 by cherry-picking fixes from #633

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Comment by MemeWizard** [comment]
@robmart @zumbak04 thanks for reviews

**Pull request merged**